src/main/webapp/scholarx/2020/index.html

```diff
@@ -54,6 +56,50 @@
     </div>
 </section>
 
+<section class="section">
+    <div class="container mt-5 mb-5" >
+        <div class="row">
+            <div class="col-md-8 offset-md-2">
+                <h4>Program Timeline</h4>
+                <ul class="timeline">
+                    <li>
+                        <a href="#">Open applications</a>
```

Comment:
Anchor tags are used to add links. Use spans/divs if there are no links.

Reply:
span wont work with this effect

Reply:
you have to write a custom css rule